hasura/metadata/databases/default/tables/public_pending_token_gifts.yaml

```diff
@@ -30,6 +30,28 @@ object_relationships:
   using:
     foreign_key_constraint_on: sender_id
 select_permissions:
+- permission:
+    allow_aggregations: true
+    columns:
+    - circle_id
+    - created_at
+    - dts_created
+    - epoch_id
+    - id
+    - recipient_id
+    - sender_id
+    - tokens
```

Comment:
Are we missing `note` col here for select permissions ? Or its a privacy concern ?

Reply:
Yes, leaving personal notes as private for now